db/migrate/20181210185019_remove_unused_columns_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveUnusedColumnsFromUsers < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :users, :sign_in_count, :integer, default: 0, null: false
```

Comment:
Thanks so much!! You did well here, add a migration and not modify the one existant. 👏